packages/context/src/objects/objects.ts

```diff
@@ -33,7 +33,7 @@ export type Action = {
     name: string;
     type: ActionType;
     outputEvent: string;
-    inputProperties?: Property[];
+    inputProperties?: Record<string, unknown>[];
```

Comment:
I assume we wanted a more specific type?  @rverma3

Reply:
Do we want a type incorporating all the different properties from every FormIO field type? 
This example below doesn't have properties that the "Select" field will use i.e. missing fields like data.values etc.  
If that's the case then we can merge all the field props from all formIo fields into one single type.  
export type FormIoInputType= {
    html: string,
    label: string,
    customClass?: string,
    refreshOnChange: boolean,
    hidden: boolean,
    modalEdit: boolean,
    conditional: unknown,
    type: string,
    key: string,
    input: boolean,
    placeholder?: string,
    prefix?: string,
    suffix?: string,
    multiple: boolean,
    defaultValue: null,
    protected: boolean,
    unique: boolean,
    persistent:boolean,
    clearOnHide:boolean,
    refreshOn?: string,
    redrawOn?: string,
    tableView: boolean,
    dataGridLabel: boolean,
    labelPosition: string,
    description?: string,
    errorLabel?: string,
    tooltip?: string,
    hideLabel: boolean,
    tabindex?: string,
    disabled: boolean,
    autofocus: boolean,
    dbIndex: boolean,
    customDefaultValue?: string,
    calculateValue?: string,
    calculateServer: boolean,
    widget: null,
    attributes?: unknown,
    validateOn: string,
    validate: unknown
    overlay: unknown
    allowCalculateOverride: boolean,
    encrypted: boolean,
    showCharCount: boolean,
    showWordCount: boolean,
    properties?: unknown,
    allowMultipleMasks: boolean,
    addons: any [],
    id: string,
}

Reply:
The idea is for custom widget developers to know how to populate action input.  I don't know if all of that is necessary, but at least some subset of it (certainly important ones like `type`, `key`, `label`).  Should probably work with @rverma3 on what should be included.